vendor/licenses/grammar/language-kotlin.txt

```diff
@@ -205,3 +204,23 @@ license: apache-2.0
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
```

Comment:
Did you add this manually? It doesn't seem to correspond to [your LICENSE file](https://github.com/nishtahir/language-kotlin/blob/f5658d73ae84ac05369c2fe5e53232e2c3301c88/LICENSE).

Reply:
Looks like my module wasn't updated before the script copied over the LICENSE. It should be good now.

Reply:
I probably missed something because I don't see what's changed.

Reply:
The link to the conversation still works for me. There hasn't been any push to your branch in the last 4 hours. The file with the license in this branch still contains your addendum, whereas the LICENSE file in your repository still doesn't.

Reply:
I see I misunderstood. I believe the addendum is coming from the [NOTICE](https://github.com/nishtahir/language-kotlin/blob/master/NOTICE) file which is in the repository. I think the script merges them into 1 file.

Reply:
Ah, sorry. That's on me then; I should know that.

@lildude I can't find the code that writes the `vendor/licenses/grammar/language-kotlin.txt` file. Is that in the closed-source licensed gem? Are we okay with the grammar repository having this notice?

Reply:
> @lildude I can't find the code that writes the vendor/licenses/grammar/language-kotlin.txt file. Is that in the closed-source licensed gem?

It's not closed source anymore - https://githubengineering.com/improving-your-oss-dependency-workflow-with-licensed/ 😁

https://github.com/github/linguist/pull/3982 will bring us up-to-date once it's merged, pending thoughts from you and @Alhadis 😉 (ref  https://github.com/github/linguist/pull/3982#issuecomment-372017975).

> Are we okay with the grammar repository having this notice?

I think so, but I'll seek some guidance from someone more knowledgable in this matter. @mlinksva is this ok?

Reply:
> It's not closed source anymore - https://githubengineering.com/improving-your-oss-dependency-workflow-with-licensed/ 😁

Ooh! I missed this :tada: 

> I think so, but I'll seek some guidance from someone more knowledgable in this matter. @mlinksva is this ok?

@lildude I'm mostly worried that we might not detect changes in the NOTICE file in the future. As far as I know, we only have a hash of the LICENSE file given by Licensee.

Reply:
The NOTICE file here is fine.

If there's a case in which licensed would fail to pick up an updated NOTICE file please open an issue at https://github.com/github/licensed/issues 🙇

Reply:
I guess with this all of the issues have been resolved, right?

Reply:
Yep, since Licensed/Licensee know how to handle the NOTICE file, we're all good :-)